src/librustdoc/clean/mod.rs

```diff
@@ -495,6 +495,12 @@ impl Item {
         self.stability.as_ref().map(|s| &s.since[..])
     }
 
+    pub fn is_non_exhaustive(&self) -> bool {
+        self.attrs.other_attrs.iter()
+            .filter(|a| a.name().as_str() == "non_exhaustive")
+            .count() > 0
```

Comment:
It may be more peformant to use [`Iterator::any`](https://doc.rust-lang.org/nightly/std/iter/trait.Iterator.html#method.any) instead of `.count() > 0`

Reply:
Fixed this.